system/conftest.py

```diff
@@ -1,23 +1,18 @@
 import pytest
-import json
 import asyncio
 import os
 from datetime import datetime
 from async_generator import async_generator, yield_
 
-from indy import payment, did
-
-from indy_vdr import set_protocol_version, ledger
+from indy_vdr import set_protocol_version
 
 from .utils import (
     pool_helper, wallet_helper, default_trustee,
-    check_no_failures, NodeHost, payment_initializer,
-    send_nym
```

Comment:
Is the “send_nym” still needed?

Reply:
Great question!
No, that import was not being used in system/conftest.py.